cinema/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 
-from .models import CinemaHall, Genre, Actor, Movie, MovieSession, Order, Ticket
+from .models import CinemaHall, Genre, Actor, \
```

Comment:
will de better if write import in format ;
from ...... import ( .........,
                             .........,
                             .........)